src/babel/transformation/transformers/other/regenerator.js

```diff
@@ -1,11 +1,69 @@
 import regenerator from "regenerator";
 import * as t from "../../../types";
+import {NodePath} from "ast-types";
 
 export var metadata = {
   group: "regenerator"
 };
 
-export function Program(ast) {
-  regenerator.transform(ast);
-  this.stop();
+export function Function(node) {
```

Comment:
This will have to be `export function Func(` (aliase keys are defined [here](https://github.com/babel/babel/blob/master/src/babel/types/alias-keys.json#L29-31)) otherwise Istanbul will throw up as it gets the global via `new Function("return this;")` which will use this function.